languages/ruby/exercises/concept/basics/.docs/introduction.md

```diff
@@ -0,0 +1,53 @@
+Ruby is a dynamic [object-oriented language]. Everything in Ruby is an [object].
```

Comment:
```suggestion
Ruby is a dynamic [object-oriented language][object-oriented-programming]. Everything in Ruby is an [object][object].
```

There is no link defined for the `[object]` reference, so one has to be added to the bottom of this file.